app/src/main/res/values/strings.xml

```diff
@@ -14,18 +15,18 @@
     <string name="arrow_description">An arrow pointing downward</string>
     <string name="creating_database">Creating database&#8230;</string>
     
-    <string name="points_statement_zero">You haven\'t earned any gold coins&#8230; time to show me what you can do!</string>
-    <plurals name="points_statement">
-        <item quantity="one">You\'ve earned &lt;b>%d gold coin&lt;/b> &#8212; nice start!</item>
-        <item quantity="other">You\'ve earned &lt;b>%d gold coins&lt;/b>.</item>
+    <string name="countries_visited_statement_zero">You haven\'t visited any countries &#8230; let\'s start with some short trips and see how you do.</string>
+    <plurals name="countries_visited_statement">
+        <item quantity="one">You\'ve visited &lt;b>%d country&lt;/b> &#8212; nice start!</item>
+        <item quantity="other">You\'ve visited &lt;b>%d different countries&lt;/b>!</item>
```

Comment:
It would be fun to select a small random subset of the visited countries and include them here, something like "You've visited 42 different countries, including France, South Africa and Canada!"

Reply:
and I could also include a few countries not yet visited, something like "You still haven't been to Mexico, Iran, or South Korea."